Tests/SonarScanner.MSBuild.TFS.Test/CoverageReportProcessorTests.cs

```diff
@@ -35,22 +34,22 @@ namespace SonarScanner.MSBuild.PostProcessor.Tests
     [TestClass]
     public class CoverageReportProcessorTests
     {
-        private Mock<ILegacyTeamBuildFactory> legacyFactoryMock;
-        private Mock<ICoverageReportConverter> converterMock;
+        private ILegacyTeamBuildFactory legacyFactoryMock;
+        private ICoverageReportConverter converterMock;
         private CoverageReportProcessor processor;
 
         [TestInitialize]
         public void TestInitialize()
         {
-            converterMock = new Mock<ICoverageReportConverter>(MockBehavior.Strict);
-            legacyFactoryMock = new Mock<ILegacyTeamBuildFactory>(MockBehavior.Strict);
-            processor = new CoverageReportProcessor(legacyFactoryMock.Object, converterMock.Object, new TestLogger());
+            converterMock = Substitute.For<ICoverageReportConverter>();
```

Comment:
Same as before, also below
```suggestion
            converter = Substitute.For<ICoverageReportConverter>();
```